server/terminal/terminal.go

```diff
@@ -0,0 +1,233 @@
+package terminal
```

Comment:
Nitpick: I think `terminal` should be under `application` packaged

Reply:
Thanks alext. I just moves those files to the `application` package. Please let me know if this is what you want to have.